src/styles/blocks/stars--0.css

```diff
@@ -0,0 +1,44 @@
+.stars--0 {
+  display: flex;
+  margin-top: 22px;
```

Comment:
According to the checklist, you should be consistent with your margins and add only top or bottom, but not both. Here you have a `margin-top` which might be inconsistent if you have also used `margin-bottom` elsewhere for similar elements.